### PR TITLE
[Buttons] Rename to `imageTitleSpace`

### PR DESCRIPTION
**Pull request by romoore**
The previous name, `imageTitleSpacing` sounds too much like a protocol name,
one that would adjust the space between things.

Amends #2563
Implements #2495

### CODE REVIEW

File: components/Buttons/src/MDCFloatingButton.m
Comment (randallli) [inline components/Buttons/src/MDCFloatingButton.m]: this one too?

**Review by randallli — DISMISSED** [review]

File: components/Buttons/src/MDCFloatingButton.m
Comment (romoore) [reply components/Buttons/src/MDCFloatingButton.m]: Done

**Comment by romoore** [comment]
Will merge against develop once the feature branch is merged.

File: components/Buttons/src/MDCFloatingButton.m
Comment (randallli) [inline components/Buttons/src/MDCFloatingButton.m:27]: internalLayoutInsets. "Spacing" provides no new information. Its an inset.

**Review by randallli — APPROVED** [review]
feel free to apply suggestion in a different PR.

File: components/Buttons/src/MDCFloatingButton.m
Comment (romoore) [reply components/Buttons/src/MDCFloatingButton.m:27]: Well, they aren't actually used as insets - I'm just reusing that data structure. If it makes more sense, I can keep the values in a non-insets struct.

Comment (romoore) [reply components/Buttons/src/MDCFloatingButton.m:27]: Changed in #2578

**Pull request merged**
